.github/PULL_REQUEST_TEMPLATE/POLISH_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Polish Notebook(s)
+about: Finalize notebooks prior to publishing or promoting
+---
+
+## Related Issues and Dependencies
+
+…
+
+## The following checklist items have been completed for each notebook in this PR:
+
+- [ ] Proofread by other team member for grammar and spelling
```

Comment:
This is cool. Can this step be made easier by using any tool like https://github.com/jupyterlab-contrib/spellchecker or https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/spellchecker/README.html?

Reply:
@oindrillac yes, everyone should use a spellcheck extension for jupyter